sentry/values.yaml

```diff
@@ -172,6 +172,9 @@ slack: {}
 #   clientSecret:
 #   verificationToken:
 
+ingress:
+  enabled: false
```

Comment:
Could you add all the possible values please ? (You could comment them for now as the ingress is disabled per default)

Reply:
Done